### PR TITLE
Add linter to get feedback on markdown formatting

### PR DESCRIPTION
**Pull request by torstenwalter**
This is just a proposal to set up simple CI to check markdown to ensure that formatting is consistent.

The good news is that it only complained about line length. I took the opportunity to put each sentence into a separate line in the README.

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        uses: actions/checkout@v2
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      - name: Lint Code Base
+        uses: github/super-linter@v3
+        env:
+          VALIDATE_ALL_CODEBASE: true
+          DEFAULT_BRANCH: true
```
Comment (onematchfox) [inline .github/workflows/linter.yml:23]:
```suggestion
DEFAULT_BRANCH: main
```

Comment (torstenwalter) [reply .github/workflows/linter.yml]: thx - fixed now.. Also updated VALIDATE_ALL_CODEBASE to false so that only changed files are checked.

**Review by scottrigby — COMMENTED** [review]
I'm all for this 👍  just will have to be rebased after https://github.com/gitops-working-group/gitops-working-group/pull/62 is merged

**Comment by torstenwalter** [comment]
@scottrigby finally rebased

**Review by scottrigby — APPROVED** [review]
@torstenwalter Thx! 💪

**Pull request merged**